templates/emails/order.crypto.processing.hbs

```diff
@@ -0,0 +1,54 @@
+Subject: Your {{pledgeAmount}} {{pledgeCurrency}} {{order.type}} to {{collective.name}} is pending
+
+<style>
+  code {
+    display: inline-block;
+    border: 1px solid #ccc;
+    background: #ddd;
+    padding: 1px 3px;
+    margin: 0px 1px;
+  }
+</style>
+
+{{> header}}
+
+{{> toplogo}}
+
+{{#if fromCollective.name}}
+<p>Hi {{fromCollective.name}},</p>
+{{else}}
+<p>Hi!</p>
+{{/if}}
+
+<p>Thank you for your crypto contribution to {{collective.name}} in the amount of {{pledgeAmount}} {{pledgeCurrency}}, made on
+  {{moment order.createdAt}}.</p>
+
+<p>Your contribution is currently pending. Once you transmit the funds,
+   and they have been confirmed as received by <a href="https://opencollective.com/{{host.slug}}">{{host.name}}</a>,
+   the amount will be credited to {{collective.name}}. </p>
+
+<p>Note that we use <a href="https://www.thegivingblock.com/">Giving Block</a>
```

Comment:
It's not necessary to mention The Giving Block.

Reply:
Thanks, I've removed this paragraph. 👍🏽